stable/dokuwiki/templates/deployment.yaml

```diff
@@ -18,6 +18,15 @@ spec:
         app: {{ template "dokuwiki.name" . }}
         chart: {{ template "dokuwiki.chart" . }}
         release: {{ .Release.Name | quote }}
+{{- if or .Values.podAnnotations .Values.metrics.enabled }}
```

Comment:
`podAnnotations` is not defined in `values.yaml` (and `values-production.yaml` if it exists).

Can you add it and also include the new variable in the README?